source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -152,21 +374,153 @@ provision the following networking API entities, represented in yaml:
     cidr: "bef0:1234:a890:5678::/64"
     enable_dhcp: yes
     gateway_ip: bef0:1234:a890:5678::1
+    ipv6_address_mode: dhcpv6_stateful
   register: ipv6_subnet
 
-- name: create a logical switch port
-  os_port:
+- name: create a router to send RA messages
+  os_router:
     cloud: "{{ cloud_name }}"
     state: present
-    name: lport1
-    network: "{{ ipv6_network.id }}"
-    fixed_ips:
-      - subnet_id: "{{ ipv6_subnet.id }}"
+    name: router0
+    interfaces:
+      - "{{ ipv6_subnet.id }}"
 ~~~~~
 
+### Configuring stateless DHCPv6 on the subnet
+
+To configure stateless dhcpv6 on the subnet, the user should provision
+the following yaml based ansible playbook snippet:
+
+~~~~~
+- name: create a network
+  os_network:
+    cloud: "{{ cloud_name }}"
+    state: present
+    name: netv6
+  register: ipv6_network
+
+- name: attach a subnet to the logical network
+  os_subnet:
+    cloud: "{{ cloud_name }}"
+    state: present
+    network_name: netv6
+    name: subnet_ipv6
+    ip_version: 6
+    cidr: bef0:1234:a890:5678::/64
+    gateway_ip: bef0:1234:a890:5678::1
+    ipv6_address_mode: dhcpv6_stateless
+  register: dhcpv6_subnet
+
+- name: create a router to send RA messages
+  os_router:
+    cloud: "{{ cloud_name }}"
+    state: present
+    name: router0
+    interfaces:
+      - "{{ dhcpv6_subnet.id }}"
+~~~~~
+
+## UI considerations
+
+The subnet ipv6_address_mode parameter should be added to the UI, since
+updates to existing subnets are not possible through the UI, and updates to the
+ipv6_address_mode parameter are not allowed, as per the
+[networking API](https://developer.openstack.org/api-ref/network/v2/?expanded=create-subnet-detail,update-subnet-detail#update-subnet). Until the UI is updated,
+the subnet should be created via ovirt-provider-ovn RESTful API or ansible.
+
+It is outside the scope of this feature to implement the UI changes that allow
+logical routers to be provisioned, and the REST API - or its ansible wrapper
+should be used instead. An ansible example is available
+[here](#configuring-stateless-dhcpv6-on-the-subnet).
+
 ## Testing
 
```

Comment:
As we learned by pain, the graphical flow via ManageIQ should be tested, too.

Reply:
Referred that testing manageIQ integration is also part of the task.